src/comp/shareButtonData/data.tsx

```diff
@@ -30,15 +24,16 @@ export type Icon = Record<IconName, ShareButtonSet>;
 export const mainIconName: IconName[] = ["Twitter", "Facebook"];
 export const subIconName: IconName[] = ["LINE", "Mail", "Pocket"];
 
+type NeededIcon = "main" | "sub" | "both";
 type UrlProps = {
   url: string;
   introduction: string;
   title: string;
 };
-export const Url = ({ url, introduction, title }: UrlProps) => {
+const ShareSet = ({ url, introduction, title }: UrlProps) => {
```

Comment:
```suggestion
const shareSet = ({ url, introduction, title }: UrlProps) => {
```
こちらは関数名なので camelCase にして、
- `ShareButtonSet` -> `ShareButton` (オブジェクトの集合じゃないのに Set はわかりにくい)
- `Icon` -> `ShareButtonSet` (オブジェクトの集合なのに単数形なのでわかりにくい)
というように名前を変えてほしい